include/dpp/json_interface.h

```diff
@@ -54,7 +54,7 @@ namespace dpp {
 		 * @param with_id Include the ID in the JSON
 		 * @return std::string JSON string version of object
 		 */
-		virtual std::string build_json(bool with_id = false) const {
+		virtual std::string build_json([[maybe_unused]] bool with_id = false) const {
```

Comment:
how can this be maybe unused when it's an optional parameter?

Reply:
Same as my other comment